src/ucp/proto/proto_init.c

```diff
@@ -248,14 +295,16 @@ ucp_proto_init_parallel_stages(const ucp_proto_init_params_t *params,
 
 ucs_status_t
 ucp_proto_common_init_caps(const ucp_proto_common_init_params_t *params,
-                           const ucp_proto_common_tl_perf_t *perf,
+                           const ucp_proto_common_tl_perf_t *tl_perf,
+                           ucp_proto_perf_node_t *const tl_perf_node,
```

Comment:
The method seems overloaded.
Is it possible to re-factor code a little? E.g. put calculation of sender overhead, transport time, receiver overhead into separate functions.

It is not obvious that the method does what it does from the signature. Guess the comment at the method declaration will help.